src/components/DomainCard.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import a from "@/../public/assets/domain/image.svg"
+import Image from 'next/image'
+
+interface props {
+    domain: {
+        id: number;
+        name: string;
+        img: string;
+    }
+}
+ 
+const DomainCard = ({domain}:props) => {
+  return (
+    <div className=' rounded-xl flex flex-col bg-white text-black '>
+        <Image className='' src={a} alt=''/>
```

Comment:
always provide alt to image and specify height and width  else build issue will occur